app/src/main/java/org/mozilla/fenix/gecko/GeckoProvider.kt

```diff
@@ -96,7 +96,8 @@ object GeckoProvider {
 
         geckoRuntime.autocompleteStorageDelegate = GeckoAutocompleteStorageDelegate(
             GeckoCreditCardsAddressesStorageDelegate(autofillStorage) {
-                context.settings().shouldAutofillCreditCardDetails
+                context.settings().shouldAutofillCreditCardDetails ||
+                    context.settings().shouldAutofillAddressDetails
```

Comment:
This isn't correct since we have both `isCreditCardAutofillEnabled` and `isAddressAutofillEnabled` We would be overriding the existing credit card autofill when they should be separate.

Reply:
Yeah, I added that before the AC bump in order to remember to add the param, but I actually forgot to change it :)